File: 8kyu_are_you_playing_banjo/solution.ex
# My solution
#
defmodule TalantDetector do
  def are_you_playing_banjo?(name) do
    first_char = String.slice(name,0..0)
    if first_char == "R" || first_char == "r" do 
      "#{name} plays banjo"
    else
      "#{name} does not play banjo"
    end
  end
end

# Better solution
#
defmodule TalantDetector do
  def are_you_playing_banjo?(name) do
    case (name |> String.first |> String.downcase) do
      "r" -> "#{name} plays banjo"
      _ -> "#{name} does not play banjo"
    end
  end
end

File: 8kyu_are_you_playing_banjo/solution.rb
# My solution
#
def are_you_playing_banjo(name)
  # Implement me!
  return "#{name} plays banjo" if name[0].upcase == "R"
  return "#{name} does not play banjo"
end


# Other solutions
#
def are_you_playing_banjo(name)
  if name.start_with? 'R', 'r'
    name + " plays banjo" 
  else  
    name + " does not play banjo" 
  end 
end
